src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -416,8 +416,8 @@ public String toString() {
     
     public static class TwitterTweet extends MessageEntry implements Runnable {
 
-        public final Semaphore ready;
-        public UserEntry user;
+        private final Semaphore ready;
+        private UserEntry user;
         public boolean writeToIndex, writeToBackend;

```

Comment:
Is this one private or public?As i can see you have modified this variable in #1133 . And can you please use one line declarations as we started solving codacy issues, don't want to add this up to the issue there again.